src/js/common/schemaform/FormApp.jsx

```diff
@@ -51,8 +51,10 @@ class FormApp extends React.Component {
     //  saved form / prefill
     // If we're in production, we'll redirect if we start in the middle of a form
     // In development, we won't redirect unless we append the URL with `?redirect`
+    const resumeForm = this.props.currentLocation.search.includes('resume');
     const devRedirect = __BUILDTYPE__ !== 'development' || this.props.currentLocation.search.includes('redirect');
-    if (isInProgress(this.props.currentLocation.pathname) && devRedirect) {
+    const goToStartPage = resumeForm || devRedirect;
+    if (isInProgress(this.props.currentLocation.pathname) && goToStartPage) {
```

Comment:
I'm guessing the idea is to link to `/path/to/form/some-page?resume` from the profile page so we can auto-load the form, yeah? We'll still need to link to a page _inside_ the form (not just the intro) for that to work, but if that's no problem, then great!